frontend/src/static/css/style.scss

```diff
@@ -453,6 +454,34 @@ header {
   }
 }
 
+.toolbar {
```

Comment:
Can you please place this whole peace at the end of same file. Because these lines will not be split into its own file anymore.